github/apps.go

```diff
@@ -32,8 +36,21 @@ type App struct {
 
 // InstallationToken represents an installation token.
 type InstallationToken struct {
-	Token     *string    `json:"token,omitempty"`
-	ExpiresAt *time.Time `json:"expires_at,omitempty"`
+	Token        *string                  `json:"token,omitempty"`
+	ExpiresAt    *time.Time               `json:"expires_at,omitempty"`
+	Permissions  *InstallationPermissions `json:"permissions,omitempty"`
+	Repositories *[]Repository            `json:"repositories,emitempty"`
+}
+
+// InstallationTokenParameters allow restricting a token's access to specific repositories.
+type InstallationTokenParameters struct {
```

Comment:
Let's please rename this to `InstallationTokenOptions` to be more consistent with the rest of the repo.

Reply:
Done.